pom.xml

```diff
@@ -266,7 +266,7 @@
         <version.jsoup>1.8.3</version.jsoup>
         <version.net.jcip>1.0</version.net.jcip>
         <version.net.shibboleth.utilities.java-support>7.1.1</version.net.shibboleth.utilities.java-support>
-        <version.org.apache.activemq.artemis>2.6.3.jbossorg-00014</version.org.apache.activemq.artemis>
+        <version.org.apache.activemq.artemis>2.6.4</version.org.apache.activemq.artemis>
```

Comment:
Does an upgrade of org.apache.qpid:proton-j come with this?

Reply:
no, there is no dependency change (no netty, nor proton-j)